docs/openapi.yaml

```diff
@@ -61,6 +66,8 @@ paths:
                   message:
                     type: string
                     example: "The email and password field is required"
+                required:
+                  - not null
```

Comment:
sao lại not null em? chỗ này mình ghi tên field mà

Reply:
đã update ạ